peps/pep-0770.rst

```diff
@@ -380,9 +383,68 @@ For projects that do contain other software components, documentation will be
 added to the Python Packaging User Guide for how to specify and maintain
 SBOM documents for Python packages in source code.
 
+There are two "camps" of projects that contain other software, those from
```

Comment:
Of course, a project can be both (I think NumPy might vendor some BLAS Fortran?)
```suggestion
There are two groups of projects that contain other software, those from
```